draft-ietf-ppm-dap.md

```diff
@@ -316,14 +316,18 @@ Batch interval:
   of the reports in the batch.
 
 Client:
-: A party that uploads a report.
+: DAP protocol role identifying a party that uploads a report. Note the
```

Comment:
```suggestion
: A DAP protocol role identifying a party that uploads a report. Note the
```

(editorial, matching structure of other items in this list)

Reply:
Sure. "The" feels a bit more natural than "A" to me.